ci/build.yml

```diff
@@ -413,6 +413,14 @@ jobs:
     - template: tell-slack-failed.yml
     - template: report-end.yml
 
+- job: compatibility_stable_protobuf
+  pool:
+    name: ubuntu_20_04
+    demands: assignment -equals default
+  steps:
+    - checkout: self
+    - bash: ci/check-protobuf-against-stable.sh
```

Comment:
```suggestion
    - bash: ci/check-protobuf-against-stable.sh
    - template: tell-slack-failed.yml
```

Reply:
Is this enough to enable Slack integration?

Reply:
Yes, assuming no typo on my end of course.